client/state/ui/editor/video-editor/reducer.js

```diff
@@ -0,0 +1,99 @@
+/**
+ * External dependencies
+ */
+import { combineReducers } from 'redux';
+
+/**
+ * Internal dependencies
+ */
+import {
+	VIDEO_EDITOR_RESET_STATE,
+	VIDEO_EDITOR_SET_POSTER_URL,
+	VIDEO_EDITOR_SHOW_ERROR,
+	VIDEO_EDITOR_SHOW_UPLOAD_PROGRESS,
+} from 'state/action-types';
+
+/**
+ * Tracks poster updated state.
+ *
+ * @param  {Object} state  Current state
+ * @param  {Object} action Action object
+ * @return {Object}        Updated state
```

Comment:
it actually looks like this returns a `Boolean`, and `Updated state` may not be the most descriptive. it's not too difficult since the function name itself documents, but often I like to at least put something like "whether or not a video poster has been updated already" if I put anything at all on the `@returns` tag.